google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -58,12 +58,26 @@ void GetBucketMetadata(storage::Client client, int& argc, char* argv[]) {
 }
 //! [get bucket metadata]
 
+//! [list objects]
+void ListObjects(storage::Client client, int& argc, char* argv[]) {
+  if (argc < 2) {
+    throw Usage{"list-objects <bucket-name>"};
+  }
+  auto bucket_name = ConsumeArg(argc, argv);
+  for (auto const& meta : client.ListObjects(bucket_name)) {
+    std::cout << "bucket_name=" << meta.bucket()
```

Comment:
I don't suppose it matters too much since this is just a sample snippet, but it seems like overkill to print out the bucket name (which will be the same) for every object.